packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -134,24 +137,50 @@ export interface SummaryElement {
      */
     readonly key: string;
 
-    // See IChannel
+    /**
+     * {@inheritDoc @fluidframework/datastore-definitions#(IChannel:interface).getAttachSummary}
+     * @param stringify - Serializes the contents of the index (including {@link IFluidHandle}s) for storage.
+     */
     getAttachSummary(
+        stringify: SummaryElementStringifier,
```

Comment:
The implementation param name is `serialize`, we should probably pick one. I'm also wondering if `SummaryElementSerializer` would be a better name, `Serializer` seems to be the standard in the repo.

Reply:
Same for summarize.

Reply:
Oops, good catch. I think they should both be `stringify`. `Serializer` is indeed a standard already, and it means something else. The function _on_ `Serializer`, which looks very much like this one, is called `stringify`.